package.json

```diff
@@ -1,19 +1,18 @@
 {
-  "author": "Shinichi Tomita <shinichi.tomita@gmail.com>",
-  "name": "jsforce",
-  "description": "Salesforce API Library for JavaScript",
+  "name": "@salto-io/jsforce",
+  "description": "Fork of jsforce with some modifications",
   "keywords": [
     "salesforce",
     "salesforce.com",
     "sfdc",
     "force.com",
     "database.com"
   ],
-  "homepage": "http://github.com/jsforce/jsforce",
+  "homepage": "https://github.com/salto-io/jsforce.git",
```

Comment:
I think the homepage can stay the same, just the repo url should move to point to the fork IMO